packages/zoe/test/unitTests/zcf/test-zcf.js

```diff
@@ -194,3 +188,202 @@ test(`zcf.getAmountMath - from issuerKeywordRecord & zcf.saveIssuer`, async t =>
+  const { moolaKit } = setup();
+  const { zcf } = await setupZCFTest();
+  // TODO: why does this not throwAsync?
+  // TODO: improve error message
+  // https://github.com/Agoric/agoric-sdk/issues/1702
+  // @ts-ignore
+  t.throws(() => zcf.saveIssuer('A', moolaKit.issuer), {
+    message: `(an object) must be a string\nSee console for error data.`,
+  });
+});
+
+test(`zcf.makeInvitation - no offerHandler`, async t => {
+  const { zcf, zoe } = await setupZCFTest();
+  const invitationP = zcf.makeInvitation(undefined, 'myInvitation');
+  const invitationIssuer = await E(zoe).getInvitationIssuer();
+  const isLive = await E(invitationIssuer).isLive(invitationP);
+  t.truthy(isLive);
+  const seat = E(zoe).offer(invitationP);
+
+  // TODO: this should not throw
+  // https://github.com/Agoric/agoric-sdk/issues/1703
+  // const offerResult = await E(seat).getOfferResult();
+  // t.is(offerResult, undefined);
+  await t.throwsAsync(() => E(seat).getOfferResult());
+});
+
+test(`zcf.makeInvitation - no-op offerHandler`, async t => {
+  const { zcf, zoe } = await setupZCFTest();
+  const invitationP = zcf.makeInvitation(() => {}, 'myInvitation');
+  const invitationIssuer = await E(zoe).getInvitationIssuer();
+  const isLive = await E(invitationIssuer).isLive(invitationP);
+  t.truthy(isLive);
+  const seat = E(zoe).offer(invitationP);
+  const offerResult = await E(seat).getOfferResult();
+  t.is(offerResult, undefined);
+});
+
+test(`zcf.makeInvitation - throwing offerHandler`, async t => {
+  const { zcf, zoe } = await setupZCFTest();
+  const handler = () => {
+    throw Error('my error message');
+  };
+  const invitationP = zcf.makeInvitation(handler, 'myInvitation');
+  const invitationIssuer = await E(zoe).getInvitationIssuer();
+  const isLive = await E(invitationIssuer).isLive(invitationP);
+  t.truthy(isLive);
+  const seat = E(zoe).offer(invitationP);
+  await t.throwsAsync(() => E(seat).getOfferResult(), {
+    message: 'my error message',
+  });
+});
+
+test(`zcf.makeInvitation - no description`, async t => {
+  const { zcf } = await setupZCFTest();
+  // @ts-ignore
+  t.throws(() => zcf.makeInvitation(() => {}), {
+    message: `invitations must have a description string: (an undefined)\nSee console for error data.`,
+  });
+});
+
+test(`zcf.makeInvitation - non-string description`, async t => {
+  const { zcf } = await setupZCFTest();
+  // TODO: improve error message
+  // https://github.com/Agoric/agoric-sdk/issues/1704
+  // @ts-ignore
+  t.throws(() => zcf.makeInvitation(() => {}, { something: 'a' }), {
+    message: `invitations must have a description string: (an object)\nSee console for error data.`,
+  });
+});
+
+test(`zcf.makeInvitation - no customProperties`, async t => {
+  const { zcf, zoe, instance, installation } = await setupZCFTest();
+  const invitationP = zcf.makeInvitation(() => {}, 'myInvitation');
+  // TODO: allow promises for payments
+  // https://github.com/Agoric/agoric-sdk/issues/1705
```

Comment:
The issue may have been revealed while writing the test, but it doesn't appear that fixing the issue will affect the test. If the test should be written differently once the issue is addressed, say so explicitly as on line 287.

Reply:
It's only a types issue, so the test is not affected.